sites/dotcom/components/Page.js

```diff
@@ -1,34 +1,6 @@
 // @flow
 import { styled } from '@guardian/guui';
 
-import {
-    mobileLandscape,
-    tablet,
-    desktop,
-    leftCol,
-    wide,
-} from '@guardian/pasteup/breakpoints';
-
-const Page = styled('div')({
-    margin: 'auto',
-    paddingLeft: 4,
-    paddingRight: 4,
-    [mobileLandscape]: {
-        paddingLeft: 24,
-        paddingRight: 24,
-    },
-    [tablet]: {
-        maxWidth: '740px',
-    },
-    [desktop]: {
-        maxWidth: '980px',
-    },
-    [leftCol]: {
-        maxWidth: '1140px',
-    },
-    [wide]: {
-        maxWidth: '1300px',
-    },
-});
+const Page = styled('div')();
```

Comment:
Can we get rid of `sites/dotcom/components/Page.js` now as it's just exporting unstyled div?

Reply:
Yeah, we could do, I guess...

However I think making `Page` the root of every component in `pages` provides a nice abstraction, a place to add page-level styling should we ever need it. Shall we keep it around, see if it's useful and ditch it if not?